commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountLocalServiceImpl.java

```diff
@@ -320,22 +320,23 @@ public CommerceAccount getGuestCommerceAccount(long companyId)
 
 		User defaultUser = userLocalService.getDefaultUser(companyId);
 
-		CommerceAccount commerceAccount = new CommerceAccountImpl();
+		CommerceAccountImpl commerceAccountImpl = new CommerceAccountImpl();
```

Comment:
Is this change necessary? It could just be a rebase error, or it could be a fix for some problem that I don't see. Let me know. If it's not necessary, please revert to the previous version.

Reply:
actually I had a strange conflict on this, as I didn't make anything on this, and also I resolved using "their" version, I'll just change this chunck to the previous version, ok?